taxi/views.py

```diff
@@ -84,6 +101,23 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:car-list")
```

Comment:
```suggestion
    success_url = reverse_lazy("taxi:driver-list")
```
The `success_url` should redirect to the `taxi:driver-list` page, not `taxi:car-list`, because after deleting a driver, the list of drivers should be shown. Fix the typo.